QualifiedTeachersApi/src/QualifiedTeachersApi/Infrastructure/EntityFramework/DictionaryValueComparer.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace QualifiedTeachersApi.Infrastructure.EntityFramework;
+
+public class DictionaryValueComparer : ValueComparer<Dictionary<string, string>>
```

Comment:
The name makes this sound like a generic dictionary comparer so I'd suggest we either name it more specifically (`DictionaryOfStringStringValueComparer `, say) or actually make it generic `DictionaryValueComparer<TKey, TValue>`

Reply:
replaced with an HasJsonConversion property builder extension